dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/expression/DefaultExpressionService.java

```diff
@@ -749,11 +750,13 @@ private <T extends IdentifiableObject> Map<String, T> getUidMap( Class<T> type,
      */
     private CommonExpressionVisitor newVisitor( ExpressionItemMethod itemMethod, ExpressionParams params )
     {
+        i18nManager.getI18n();
```

Comment:
Is this needed to force some initialization, or can be removed?

Reply:
Oops, good catch @maikelarabori. This was a test statement I used to measure the performance difference with and without the fix, and I forgot to take it out. (It would have defeated the speed-up!) Taken out now. Thank you!!!